BEIMA.Backend/MongoService/MongoConnector.cs

```diff
@@ -0,0 +1,192 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using MongoDB.Driver;
+using MongoDB.Bson;
+
+namespace BEIMA.Backend.MongoService
+{
+    public sealed class MongoConnector
```

Comment:
I think it's important to have class headers as well, for maintainability.

Reply:
Fixed b8e46ae